content/en/network_monitoring/devices/config_management.md

```diff
@@ -0,0 +1,186 @@
+       ## The namespace should match namespaces of devices being monitored
+       namespace: default
+       ## @param min_collection_interval - integer - optional - default: 900 (15 minutes)
+       min_collection_interval: 900
+       ## @param ssh - object - optional
+       ## Global SSH configuration that applies to all device instances unless 
+       # overridden at the device level. 
+       ssh:
+         ## @param timeout - duration - optional - default: 30 (seconds)
+         ## Maximum time for the SSH client to establish a TCP connection.
+         timeout: 30
+         ## @param known_hosts_path - string - required (unless insecure_skip_verify is true)
+         ## Path to the known_hosts file containing public keys of servers to 
+         # verify the identity of remote hosts. Required for secure connections.
+         known_hosts_path: /path/to/known_hosts
+         ## @param insecure_skip_verify - boolean - optional - default: false
+         ## Skip host key verification. This is INSECURE and should only be used
+         ## for development/testing purposes.
+         insecure_skip_verify: false 
+      instances:
+        ip_address - string - required
+        ## The IP address of the network device to collect configurations from.
+        ip_address: <IP_ADDRESS>
+        ## @param profile - string - optional
+        ## The device profile name that defines how to collect configurations.
+        ## Examples: "cisco-ios", "junos"
+        ## If not specified, the agent will attempt to auto-detect the device type.
+        profile: <PROFILE_NAME>
+        ## @param auth - object - required
+        ## Authentication credentials to connect to the network device.
+        auth:
+          ## @param username - string - required
+          ## Username to authenticate to the network device.
+          username: <USERNAME>
+          ## @param password - string - required (if private_key_file is not provided)
+          ## Password to authenticate to the network device.
+          ## Used as a fallback after private key authentication if both are provided.
+          password: <PASSWORD>
+          ## @param private_key_file - string - optional
+          ## Path to the SSH private key file for authentication.
+          ## At least one of password or private_key_file must be provided.
+          private_key_file: /path/to/private_key
+   ```
+
+2. Optionally, if your devices require specific SSH algorithms, use the following configuration:
+
+   ```yaml
+   init_config:
+     ## @param ciphers - list of strings - optional
+     ## List of SSH encryption ciphers to use for the connection.
+     ## If not specified, the SSH library will use its default ciphers.
+     ciphers: [aes128-gcm@openssh.com, aes128-ctr, aes192-ctr]
+     key_exchanges: [diffie-hellman-group14-sha256, ecdh-sha2-nistp256]
+     host_key_algorithms: [ssh-ed25519]
+   ```
+
+   See the full [configuration file][7] for more details.
```

Comment:
This link doesn't open an actual file, it's just a folder. @git-thuerk-done I think you already mentioned it's coming soon, but flagging just in case.